### PR TITLE
[#113] Resolve working directory path

### LINKED ISSUE #113: Dunner `dir` is not correctly set
**Bug**
Dunner `dir` is not correctly set when we define it in `.dunner.yaml`

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a `.dunner.yaml`
2. Add a task which has a step to echo `$PW`
3. Define `dir` as `/go/`
2. See if the working directory is `/go` or `/dunner/go`

**Expected behavior**
It should be `/go` not `/dunner/go`

**Screenshots**
Screenshot from 2019-07-20 00-15-20

### PR DESCRIPTION
**Pull request by ayushjn20**
Identify and set the working directory path as an absolute path or a relative path. Resolves #113

### CODE REVIEW

File: pkg/dunner/dunner.go [hunk 1]
```diff
@@ -51,6 +51,10 @@ func Do(_ *cobra.Command, args []string) {
 func ExecTask(configs *config.Configs, taskName string, args []string) {
 	var async = viper.GetBool("Async")
 	var wg sync.WaitGroup
+
+	if _, exists := configs.Tasks[taskName]; !exists {
+		log.Errorf("dunner: task '%s' does not exist", taskName)
```
Comment (apoorvam) [inline pkg/dunner/dunner.go]: This should become a part of validation and not during dunner execution.

File: pkg/docker/docker.go [hunk 2]
```diff
@@ -102,7 +102,11 @@ func (step Step) Exec() (*[]Result, error) {
 
 	var containerWorkingDir = containerDefaultWorkingDir
 	if step.WorkDir != "" {
-		containerWorkingDir = filepath.Join(hostMountTarget, step.WorkDir)
+		if step.WorkDir[0] == '/' {
```
Comment (apoorvam) [inline pkg/docker/docker.go]: Please add unit tests for this.

File: pkg/dunner/dunner.go (hunk 1 above)
Comment (ayushjn20) [reply pkg/dunner/dunner.go]: `taskName` is given by the CLI and not the dunner file. I believe validation is done for the dunner file. Could you please explain with file and line number, where should the check be implemented?

File: pkg/docker/docker.go (hunk 2 above)
Comment (ayushjn20) [reply pkg/docker/docker.go]: Done

Comment (agentmilindu) [reply pkg/docker/docker.go]: @apoorvam Can you approve the PR?

**Pull request merged**